### DISCUSSION TITLE
熊貓要確定欸

### DISCUSSION THREAD

**Discussion by AngusGuei** [1101-GS3073-C/Python-Community-GS3073-C#433] [General]
foodpanda 將對基隆、雙北、桃園、新竹美食與生鮮雜貨外送，每筆訂單加收所謂的「平台費」。
痾...這就是自斷手腳嗎 太狠了

https://technews.tw/2022/06/06/foodpanda-platform-fee/

**Comment by YuHengChen54** [comment]
這幾天訂foodpanda的時候也有發現，硬生生要多收3塊真的很不爽欸...

**Comment by BingRong1217** [comment]
外送平台的費用確實是一筆蠻大的負擔

**Comment by Ben6uo** [comment]
原本食商家的癈用就很高了 還這樣搞喔?

**Comment by brian431** [comment]
如果不是真的必要，他們也不會怎麼做吧？所以我想大家也不必太過在意。
